src/modules/auth/component/Login/Login.tsx

```diff
@@ -76,41 +68,56 @@ export function Login() {
       >
         <Grid container spacing={2}>
           <Grid item sm={12}>
-
-            <form onSubmit={handleSubmit}>
+            <form onSubmit={handleSubmit(onSubmit)}>
               <Grid container spacing={6}>
                 <Grid item sm={12}>
-                  <TextField
-                    label="Email"
-                    placeholder="email@gmail.com"
+                  <Controller
                     name="email"
-                    type="email"
-                    size="small"
-                    fullWidth
-                    value={userData.email}
-                    onChange={handleChange}
+                    control={control}
+                    rules={{ required: "Email is required" }}
+                    render={({ field }) => (
+                      <TextField
+                        label="Email"
+                        placeholder="email@gmail.com"
+                        type="email"
+                        size="small"
+                        fullWidth
+                        {...field}
+                      />
+                    )}
                   />
                 </Grid>
                 <Grid item sm={12}>
-                  <TextField
-                    label="Password"
-                    placeholder="********"
+                  <Controller
                     name="password"
-                    type="password"
-                    size="small"
-                    fullWidth
-                    value={userData.password}
-                    onChange={handleChange}
+                    control={control}
+                    rules={{ required: "Password is required" }}
+                    render={({ field }) => (
+                      <TextField
+                        label="Password"
+                        placeholder="********"
+                        type="password"
+                        size="small"
+                        fullWidth
```

Comment:
explicitly show that this is a boolean value

Reply:
done